src/main.js

```diff
@@ -33,9 +25,18 @@ let load = base.loader({
     }
   },
 
+  monitor: {
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => base.monitor({
+      project: cfg.monitor.component,
```

Comment:
ok, I think having it named "process" is a little misleading as this service does not have multiple processes that we need to identify, it's a separate instance of the same root project (taskcluster-treeherder in this case)

So I think that it should be addressed how one uses the monitor tool when:
1. you have multiple deployments of the same root project (such as a production and staging version)
2. You have multiple processes for the same deployment (such as all the processes for taskcluster-queue)
3. You have a combination of 1 and 2